### PR TITLE
feat: add modal info and change layout

### LINKED ISSUE #57: [Feature Request] About Us Link
# About Us Feature
When user visit the [website](https://colorful-quran.pages.dev), they didn't know about this app, who made this app, etc.
So I propose there is link with question icon (?) on home page (may be on top right). When it's clicked, the pop up appear.

# Content
Inside popup, should includes:
- about us, simple description about this app, app version
- contributors, we can use this [github api](https://api.github.com/repos/kodepintar/colorful-quran/contributors)
- link to this repo (user can submit issue here)
- change log
- license

### PR DESCRIPTION
**Pull request by sangvictim**
i think its done.

it will close #57  [About Us Link](https://github.com/kodepintar/colorful-quran/issues/57)

### CODE REVIEW

File: src/component/Modal.svelte [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <script lang="ts">
+	export let Title:string;
```
Comment (axmad386) [inline src/component/Modal.svelte:2]: Dibikin cameCase aja gmn gan?
Props atau varibel lebih baik camelCase
Misal surahDetail, title, subTitle
Kalau PascalCase biasnya buat penamaan class atau component misa TajweedView

**Review by axmad386 — CHANGES_REQUESTED** [review]
Cuman standarisasi

File: src/component/Modal.svelte (hunk 1 above)
Comment (sangvictim) [reply src/component/Modal.svelte:2]: ok tak revisi

File: .chglog/CHANGELOG.tpl.md [hunk 2]
```diff
@@ -1,30 +1,41 @@
 {{ if .Versions -}}
```
Comment (axmad386) [inline .chglog/CHANGELOG.tpl.md:1]: iki file keupdate formatermu?
tak revert ya

Comment (axmad386) [reply .chglog/CHANGELOG.tpl.md]: done

**Review by axmad386 — APPROVED** [review]
OK

**Comment by axmad386** [comment]
thanks @sangvictim

**Pull request merged**